llvm/utils/TableGen/DXILEmitter.cpp

```diff
@@ -233,16 +255,9 @@ static std::string getParameterKindStr(ParameterKind Kind) {
   llvm_unreachable("Unknown llvm::dxil::ParameterKind enum");
 }
 
-/// Return a string representation of OverloadKind enum that maps to
```

Comment:
Seems this comment is still valid but has now been lost.  Is this intentional?

Reply:
> Seems this comment is still valid but has now been lost. Is this intentional?

It was unintentional. Thanks for flagging it.